cmake/projects/benchmark/hunter.cmake

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2017, Giuseppe Roberti
+# All rights reserved.
+
+# !!! DO NOT PLACE HEADER GUARDS HERE !!!
+
+include(hunter_add_version)
+include(hunter_cacheable)
+include(hunter_download)
+include(hunter_pick_scheme)
+include(hunter_cmake_args)
+
+hunter_add_version(
+    PACKAGE_NAME
+    benchmark
+    VERSION
+    "1.1.0-hunter"
+    URL
+    "https://github.com/piribes/benchmark/archive/v1.1.0-hunter.tar.gz"
```

Comment:
Created https://github.com/hunter-packages/benchmark

I've added you to team (check your email). Also please read this:
* https://docs.hunter.sh/en/latest/creating-new/cmake-dependencies.html#maintenance